app/views/admin/editions/_standard_fields.html.erb

```diff
@@ -64,17 +64,24 @@
   </div>
 
   <div class="govuk-!-margin-bottom-8 js-locale-switcher-field">
-    <% if Flipflop.govspeak_visual_editor? && current_user.can_see_visual_editor_private_beta? %>
+    <% if Flipflop.govspeak_visual_editor? && current_user.can_see_visual_editor_private_beta? && form.object.visual_editor %>
       <%= render "components/visual_editor", {
         label: {
           text: "Body" + "#{' (required)' if form.object.body_required?}",
           heading_size: "m",
         },
         name: "edition[body]",
+        id: "edition_body",
         value: edition.body,
+        rows: 20,
         error_items: errors_for(edition.errors, :body),
         right_to_left: form.object.translation_rtl?,
-        id: "edition_body",
+        data_attributes: {
+          image_ids: edition.images.map { |img| img[:id] }.to_json,
```

Comment:
Is this in the scope of this PR?

Reply:
The data attributes no, but we are now passing them down because we need to use them in the conditional rendering of the govspeak editor (from within the visual editor) 🥲😑☠️ 
And that we decided to do (instead of rendering the textarea as before) because otherwise when we exit the visual editor, we lose the preview functionality.